index.html

```diff
@@ -24,6 +24,18 @@ <h1>Astoria Digital Pattern Library</h1>
     </header>
 
 
+    <nav>
+
+        <h6>Content</h6>
```

Comment:
Heading levels need to be in sequence, so an `h6` can only exist if it comes after an `h5`, which needs to come after an `h4`, etc etc. Headings are semantic that way, and have structural meaning.

If this was done for style reasons, either use a non-heading markup, like `strong`, or create a class that specifically styles the kind of thing you want to make.

It's also generally not done to put a heading inside a `nav` tag, since the heading isn't itself part of the navigation (those are generally links). Since there's no navigation content here, I don't think the `nav` tag is appropriate here.

Reply:
Thanks for catching this, I totally should have known better!

Reply:
Hey @aniagaska I made a few adjustments to the code; hope you don't mind! You can take a look at what I did here 3396aa78cd2cd96ce7b1339dabbc9dbbd3c287c0. I'm otherwise happy with this, so if it looks good to you go ahead a merge!